src/components/TaskPane/TaskPane.js

```diff
@@ -121,6 +128,14 @@ export class TaskPane extends Component {
                 <li className="mr-links-green-lighter">
                   <OwnerContactLink {...this.props} />
                 </li>
+
+                {isManageable && !this.props.inspectTask && (
+                  <li>
+                    <button className="mr-text-current" onClick={() => this.props.history.push(taskInspectRoute)}>
+                      Inspect
```

Comment:
We need the `Inspect` label to be an internationalized message (using FormatMessage) so that it can be translated. You can either create a Messages.js file for TaskPane, as it doesn't have one yet, or you could extract the inspect link generation code and isManageable check into its own InspectLink component and add the Messages into that component.

Otherwise this looks good.